src/client/app/model/arrow.ts

```diff
@@ -1,23 +1,24 @@
 import { Point } from './geometry';
+import { Card } from './card';
 import { CollaborativeObjectModel } from './collaborative-object-model';
 
 export class Arrow extends CollaborativeObjectModel {
   protected static modelName = 'Arrow';
```

Comment:
Unused?

Reply:
The modelName field? It's used in the superclass in the registerModel function. I'm just overriding it here.